gradle.properties

```
@@ -1,4 +1,4 @@
-version=7.1.1
+version=7.1.2-rc5
```

```suggestion
version=7.1.2
```
To commit right before merge